main.m

```diff
@@ -0,0 +1,3 @@
+clear();
+data = Data("IMU_Data_for_assignment/**/*.dat", 400, 50);
+writetable(data.IMU_data, "output.xlsx")
```

Comment:
Code analyser shows this takes about half the time needed for the script to run should be flagged